src/examples/java/com/amazonaws/crypto/examples/DataKeyCaching/LambdaDecryptAndWrite.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except
+ * in compliance with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.amazonaws.crypto.examples.kinesisdatakeycaching;
```

Comment:
should examples (anything inside com.amazonaws.crypto.examples package) go in src/test or totally new maven module?
Not a good idea to keep example code in final sdk code. It can be separate for references purposes.
This gives flexibility on users choosing what to add as a dependency.

being 'examples', it gives future developers an idea of keeping test codes, utils or some insecured code or doing unwanted logging which could reveal more than we expect.

thoughts?

Reply:
This is a good point. We should probably move this stuff into a different repository and just reference it from README.md.